lib/fathead/diveintopython/parse.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import codecs
+from bs4 import BeautifulSoup
+from tidylib import tidy_document
+
+URL_ROOT = 'http://www.diveintopython3.net/'
+DOWNLOADED_HTML_PATH = 'download/diveintopython3-master'
+
+class PythonData(object):
+    """
+    Object responsible for loading raw HTML data from Python docs:
+    """
+    def __init__(self, file):
+        """
+        Initialize PythonData object. Load data from HTML.
+        """
+        self.HTML = ""
+        self.FILE = file
+        self.load_data()
+
+    def load_data(self):
+        """
+        Open the HTML file and load it into the object.
+
+        """
+        with codecs.open(self.FILE, 'r', encoding='utf-8', errors='ignore') as data_file:
```

Comment:
@gaulrobe, I got unicode errors without it for files having the byte order mark or other various characters. Going to test it with the regular `open` too next.